src/command_modules/azure-cli-natgateway/azure_bdist_wheel.py

```diff
@@ -0,0 +1,54 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+from distutils import log as logger
+import os.path
+
+from wheel.bdist_wheel import bdist_wheel
+class azure_bdist_wheel(bdist_wheel):
```

Comment:
@marstr and @tjprescott, if we end up consolidating the command modules, would we be able to remove redundant definitions of this class and instances of this file?